### DISCUSSION TITLE
<title> how to delete my organazation from my profile which have been removed

### DISCUSSION THREAD

**Discussion by chosenone75** [community/community#58617] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I want to delete my organ MY1314, but can't figure out how to do it, after I click the MY1314 from my profile, it tells me it is a deleted organ, any suggestions?

**Comment by the-officialjosh** [comment]
You could try this I got from another issue maybe it solves this problem

1. Log in to your GitHub account.
2. Go to your profile by clicking on your profile picture in the top right corner of the page.
3. In the left sidebar, click on "Organizations."
4. Scroll down to the "Your organizations" section.
5. Find the organization "MY1314" in the list.
6. Click on the organization's name to access its profile.
7. On the organization's profile page, look for an option or button that says "Leave" or "Remove."
8. Click on "Leave" or "Remove" to remove yourself from the organization.

By leaving or removing yourself from the organization, it should no longer appear in your profile.

If you're unable to find the "Leave" or "Remove" option, it's possible that you've already been successfully removed from the organization, but it still appears due to caching or other technical reasons. In that case, you may need to wait for some time or clear your browser cache to see the organization removed from your profile.

Try this if it works

**Reply by chosenone75** [reply]
actually I cant find the orgin in "Your organizations" section, it's wired.

**Comment by AlbinXXX** [comment]
Log in to your GitHub account, assuming your secret code and credentials are at the ready.

1. Locate and click on your profile picture in the top right corner of the page to access your profile, just like a seasoned explorer finding their way.
2. In the left sidebar, seek out the section called "Organizations" and kindly give it a gentle tap.
3. Gradually scroll down the page until your eyes land on the segment titled "Your organizations." This is where the organization in question should reside.
4. Now, with utmost care and precision, search for the name "MY1314" within the list of organizations.
5. If you have the honor of being the organization owner, rejoice! You can confidently click on the organization's name to access its profile.
6. Upon entering the organization's profile page, carefully scrutinize the interface for a majestic option or button that triumphantly declares "Leave" or "Remove."
7. Here comes the catch: If you don't possess the coveted title of organization owner, alas, you won't have the authority to bid adieu to it. In this case, you'll need to reach out to the organization owner or an admin and politely request their assistance in removing you from the organization. However, if you do have the power of ownership, seize the moment and click on "Leave" or "Remove" to free yourself from the organization's clutches.

Voilà! By taking this action, the organization should gracefully bow out from your profile, as if it were never there to begin with.
